server/controllers/process.go

```diff
@@ -165,7 +167,7 @@ func ProcessRun(rw io.ReadWriteCloser, c *api.Context) error {
 
 	ev, err := url.ParseQuery(c.Header("Environment"))
 	if err != nil {
-		return helpers.CodeError(rw, -1, err)
+		return helpers.CodeError(rw, 255, err)
```

Comment:
A return of -1 broke the code/error scanner which didn't expect the `-` character.

A return code of -1 is invalid, so I assumed the code scanner was correct and picked a new valid number